mlos_bench/db/db_schema.sql

```diff
@@ -2,37 +2,137 @@
 -- Copyright (c) Microsoft Corporation.
 -- Licensed under the MIT License.
 --
+
```

Comment:
Required whitespace to avoid the `licenseheaders` tool from removing the comment.